[2022-05-31 19:32:26,059] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: s3_to_redshift.create_table manual__2022-05-31T19:32:24.550784+00:00 [queued]>
[2022-05-31 19:32:26,076] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: s3_to_redshift.create_table manual__2022-05-31T19:32:24.550784+00:00 [queued]>
[2022-05-31 19:32:26,077] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 19:32:26,077] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2022-05-31 19:32:26,077] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 19:32:26,093] {taskinstance.py:1262} INFO - Executing <Task(PostgresOperator): create_table> on 2022-05-31 19:32:24.550784+00:00
[2022-05-31 19:32:26,107] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 's3_to_redshift', 'create_table', 'manual__2022-05-31T19:32:24.550784+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/s3_to_redshift_dag.py', '--cfg-path', '/tmp/tmpy30001fd', '--error-file', '/tmp/tmpvznr5ynt']
[2022-05-31 19:32:26,108] {standard_task_runner.py:77} INFO - Job 3: Subtask create_table
[2022-05-31 19:32:26,104] {standard_task_runner.py:52} INFO - Started process 8738 to run task
[2022-05-31 19:32:26,144] {logging_mixin.py:109} INFO - Running <TaskInstance: s3_to_redshift.create_table manual__2022-05-31T19:32:24.550784+00:00 [running]> on host 96846966a853
[2022-05-31 19:32:26,189] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=s3_to_redshift
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-05-31T19:32:24.550784+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-31T19:32:24.550784+00:00
[2022-05-31 19:32:26,197] {base.py:79} INFO - Using connection to: id: redshift. Host: redshift-cluster-1.ctwsihrera0y.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: ***, extra: {}
[2022-05-31 19:32:42,630] {dbapi.py:225} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS trips (
trip_id INTEGER NOT NULL,
start_time TIMESTAMP NOT NULL,
end_time TIMESTAMP NOT NULL,
bikeid INTEGER NOT NULL,
tripduration DECIMAL(16,2) NOT NULL,
from_station_id INTEGER NOT NULL,
from_station_name VARCHAR(100) NOT NULL,
to_station_id INTEGER NOT NULL,
to_station_name VARCHAR(100) NOT NULL,
usertype VARCHAR(20),
gender VARCHAR(6),
birthyear INTEGER,
PRIMARY KEY(trip_id))
DISTSTYLE ALL;, parameters: None
[2022-05-31 19:32:43,272] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=s3_to_redshift, task_id=create_table, execution_date=20220531T193224, start_date=20220531T193226, end_date=20220531T193243
[2022-05-31 19:32:43,298] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-31 19:32:43,345] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
